spec/factories/service_providers.rb

```diff
@@ -1,7 +1,7 @@
 FactoryGirl.define do
   factory :service_provider do
     sequence(:friendly_name) { |n| "test-service_provider-#{n}" }
-    sequence(:issuer) { |n| "test-service_provider-#{n}" }
+    sequence(:issuer) { |n| "urn:gov:gsa:SAML:2.0.profiles:sp:sso:DEPT:APP-#{n}" }
```

Comment:
> I really don't think we should force SPs to change since there would be no continuity? 

I don't see a way around this; if the current format does not conform to what we have prescribed, they should update it, regardless if we enforce it via app validations or face to face communication.